common/djangoapps/util/tests/test_codejail_includes.py

```diff
@@ -0,0 +1,24 @@
+# lint-amnesty, pylint: disable=missing-module-docstring
+
+import unittest
+
+import eia
+import loncapa
+from verifiers import draganddrop
+
+
+class Test_CodeJail_Includes(unittest.TestCase):
+    """ tests for  codejail includes"""
+
+    def test_loncapa(self):
+        random_integer = loncapa.lc_random(2, 60, 5)
+        self.assertLessEqual(random_integer, 60)
```

Comment:
Thanks for adding these new tests!

Since we use pytest now, you can write all these test cases using normal boolean expressions and the `assert` keyword. 

For example, instead of:
```python
self.assertLessEqual(x, y)
self.assertIn(my_item, my_list)
self.assertFalse(my_boolean)
```

you can just write:
```python
assert x <= y
assert my_item in my_list
assert not my_boolean
```